recipes/mox/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: mox
+  version: {{ version }}
+
+source:
+  fn: mox-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/m/mox/mox-{{ version }}.tar.gz
+  md5: 6de7371e7e8bd9e2dad3fef2646f4a43
+
+build:
+  number: 0
+  skip: true  # [py3k]
+  script: python setup.py install
+
+requirements:
+  build:
+    - python <3
+
+  run:
+    - python <3
```

Comment:
If you add `< 3` in both `build` and `run` the binary loses the build string. Also, since you are skipping py3k above, there is no need for this here and above.